closed/src/java.base/share/classes/sun/security/ec/NativeECKeyPairGenerator.java

```diff
@@ -192,13 +211,30 @@ public void initialize(AlgorithmParameterSpec params, SecureRandom random)
         }
 
         // Not all known curves are supported by the native implementation
-        ECKeyPairGenerator.ensureCurveIsSupported(ecSpec);
+        try{
```

Comment:
Missing space before `{`.
Is there a way to do this without throwing and then ignoring the exception?

Reply:
I did update this block to the following:
```
if (!(NativeECUtil.isBrainpoolP512r1(ecSpec))) {
            ECKeyPairGenerator.ensureCurveIsSupported(ecSpec);
        }
```
This should perform this check when we are not using brainpoolP512r1 as we are going to send this operation into openssl.

Reply:
Please remove the redundant parentheses and update the preceding comment so it ends with a period.

Reply:
Removed redundant parentheses as suggested.